File: UNIDAD 1/00_SOA_SIN_BDD/002. REST_JAVA_CONUNI_GR10/02. CLIMOV/app/src/main/java/ec/edu/monster/modelo/Movimiento.kt
package ec.edu.monster.modelo

import com.google.gson.annotations.SerializedName
import java.util.Date

data class Movimiento(
    val cuenta: String,
    @SerializedName("nroMov") val nroMov: Int,
    val fecha: Date,
    val tipo: String,
    val accion: String,
    val importe: Double
)

File: UNIDAD 1/01_SOA_CON_BDD/001. SOAP_DOTNET_EUREKABANK_GR10/01. SERVIDOR/EurekaBankService/EurekaBankService/Models/Movimiento.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace EurekaBankService.Models
{
    public class Movimiento
    {
        public string Cuenta { get; set; }
        public int NroMov { get; set; }
        public DateTime Fecha { get; set; }
        public string Tipo { get; set; }
        public string Accion { get; set; }
        public double Importe { get; set; }
    }
}
